### PR TITLE
Migration support center article

### LINKED ISSUE #1065: Migrate article: Project Support Center
Migrate the content from this article to fit the documentation portal format https://help.aiven.io/en/articles/3688386

(Assign this issue to yourself before you start, and remember to mention the issue number when you open your pull request)

### PR DESCRIPTION
**Pull request by staceysalamon-aiven**
# What changed, and why it matters
Resolves #1065. Migrating and updating content on how to create, track and add followers to support tickets in Aiven Console.

### CODE REVIEW

File: docs/platform/howto/project-support-center.rst [hunk 1]
```diff
@@ -0,0 +1,35 @@
+Get Support in the Aiven Console
```
Comment (laysauchoa) [inline docs/platform/howto/project-support-center.rst:1]:
```suggestion
Get support in the Aiven console
```

File: docs/platform/howto/project-support-center.rst [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+====================================
+
+Aiven is committed to providing excellent and responsive support to all of our customers. We are constantly monitoring our services 24/7 and rapidly addressing any anomalies.
+
+In addition to our standard support, we offer three additional tiers that include faster response times and phone support. For more information about support plans, please see `Aiven support details page <https://aiven.io/support-services>`_ or contact sales@aiven.io to discuss premium support.
```
Comment (laysauchoa) [inline docs/platform/howto/project-support-center.rst:6]:
```suggestion
In addition to our standard support, we offer three additional tiers that include faster response times and phone support. For more information about support plans, please see `Aiven support details page <https://aiven.io/support-services>`_ or contact ``sales@aiven.io`` to discuss premium support.
```

File: docs/platform/howto/project-support-center.rst (hunk 1 above)
Comment (staceysalamon-aiven) [reply docs/platform/howto/project-support-center.rst:1]: Thanks for changing the support so this is sentence case. :) I had thought Aiven Console was the proper name (i.e., should be in title case)?

File: docs/platform/howto/project-support-center.rst [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+Add followers to an open ticket
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If other users want to track and manage a ticket, the owner can add them to an open ticket by the owner of the ticket:
+
+#. Click **Support** to open the Project Support Center.
+#. Find the ticket you want to add users to in the list and click **Add follower**. 
+#. Select the user's email and click **Add follower**.
+
+You can also add users to a ticket by CCing them into the support ticket email.
```
Comment (laysauchoa) [inline docs/platform/howto/project-support-center.rst:35]: Can you find a replacement for CCing?

File: docs/platform/howto/project-support-center.rst (hunk 1 above)
Comment (laysauchoa) [reply docs/platform/howto/project-support-center.rst:1]: Correct :) Only the word Aiven should be in uppercase and the initial sentence.

Comment (staceysalamon-aiven) [reply docs/platform/howto/project-support-center.rst:1]: Sorry, I meant Console should also be uppercase according to our internal Official Aiven product name guidelines. (Will share in DM.)

Comment (laysauchoa) [reply docs/platform/howto/project-support-center.rst:1]: Good point. If you are using in a text, you can use as follows: `Aiven Console <https://console.aiven.io>`_ so it goes with our guidelines. You can open an Issue to discuss whether we should replace in the titles as well to follow the guidelines. Maybe @TibsAtWork can help us out here.

**Review by ftisiot — APPROVED** [review]
Hi @staceysalamon-aiven ,
I made a couple of really minor changes but the PR looks good to me. Quick question for confirmation... are you ok to merge here both the new support center article and also the updates on DataDog?

**Comment by staceysalamon-aiven** [comment]
> Hi @staceysalamon-aiven , I made a couple of really minor changes but the PR looks good to me. Quick question for confirmation... are you ok to merge here both the new support center article and also the updates on DataDog?

Hey, thanks for looking at this. There should not be any changes for the Datadog article here and I had already fixed things like the CCing (changed it to read "copying them into"). 

@harshini-rangaswamy It looks like this commit (37699352be2ff7817c3823788e14c708b1e43fa9) updated this support center article branch?

**Comment by harshini-rangaswamy** [comment]
> > Hi @staceysalamon-aiven , I made a couple of really minor changes but the PR looks good to me. Quick question for confirmation... are you ok to merge here both the new support center article and also the updates on DataDog?
> 
> Hey, thanks for looking at this. There should not be any changes for the Datadog article here and I had already fixed things like the CCing (changed it to read "copying them into").
> 
> @harshini-rangaswamy It looks like this commit ([3769935](https://github.com/aiven/devportal/commit/37699352be2ff7817c3823788e14c708b1e43fa9)) updated this support center article branch?

@staceysalamon-aiven that is quite strange and I am not sure how this affected your branch. This is my pr ([1460](https://github.com/aiven/devportal/pull/1460)), I dont have any other files in here apart from the Datadog one. You see the changes, probably because this branch was merged with the main? Would a rebase help here?

**Review by ftisiot — APPROVED** [review]
I rebased since I saw some weirds change in the latest PR. I believe we're ready to merge now

**Comment by laysauchoa** [comment]
Hi @staceysalamon-aiven maybe you did not see those commits in your branch because it was not rebased it.

**Comment by staceysalamon-aiven** [comment]
> Hi @staceysalamon-aiven maybe you did not see those commits in your branch because it was not rebased it.

No, I always rebase but something weird happened here.

**Comment by dewan-ahmed** [comment]
thanks for this contribution 🙌🏽 

I didn't see the entries on the migration sheet so I added them.

**Pull request merged**